Source/WebDriver/Session.h

```diff
@@ -27,12 +27,18 @@
 
 #include "Actions.h"
 #include "Capabilities.h"
+#include "SessionHost.h"
+#if ENABLE(WEBDRIVER_BIDI)
```

Comment:
Please move the new `#if ENABLE` section to after unguarded headers.

Reply:
Moved.